pkg/utils.py

```diff
@@ -258,16 +258,33 @@ def process_test(
     shortener = pyshorteners.Shortener(timeout=10)
     merged_df["buildUrl"] = merged_df["uuid"].apply(
         lambda uuid: (
-            shortener.tinyurl.short(buildUrls[uuid])
+            shorten_url(shortener, buildUrls[uuid])
             if options["convert_tinyurl"]
             else buildUrls[uuid]
-        )  # pylint: disable = cell-var-from-loop
+        )
+
+        # pylint: disable = cell-var-from-loop
     )
     #save the dataframe
     output_file_path = f"{options['save_data_path'].split('.')[0]}-{test['name']}.csv"
     match.save_results(merged_df, csv_file_path=output_file_path)
     return merged_df, metrics_config
 
+def shorten_url(shortener: any, uuids: List[str]) -> str:
```

Comment:
```suggestion
def shorten_url(shortener: any, buildUrls: List[str]) -> str:
```
Isn't a uuid supposed to be mapped with just one buildUrl?

Reply:
See: https://github.com/cloud-bulldozer/orion/pull/66#discussion_r1767042759